modules/con_notifications-configuration.adoc

```diff
@@ -0,0 +1,100 @@
+:_mod-docs-content-type: CONCEPT
+[id="notifications-configuration_{context}"]
+= NotificationsConfiguration parameter
+
+The `NotificationsConfiguration` parameter is a Kubernetes custom resource (CR) that manages notifications in a Kubernetes cluster. In {gitops-title}, the `NotificationsConfiguration` (CR) allows users to add Templates, Triggers, Services, and Subscriptions resources to an Argo CD Notifications ConfigMap.
+
+When you create a cluster in {gitops-title}, a `NotificationsConfiguration` (CR) is created by default with the name `default-notifications-configuration`.
```

Comment:
When you create a cluster in {gitops-title} with **notifications enabled**.

Reply:
Fixed.